test_unstructured/partition/test_doc.py

```diff
@@ -161,61 +159,6 @@ def test_partition_doc_from_filename_prefers_metadata_last_modified_when_provide
     assert all(e.metadata.last_modified == metadata_last_modified for e in elements)
 
 
-def test_partition_doc_from_file_suppresses_last_modified_from_file_by_default(mocker: MockFixture):
-    modified_date_on_file = "2029-07-05T09:24:28"
-    mocker.patch(
-        "unstructured.partition.doc.get_last_modified_date_from_file",
-        return_value=modified_date_on_file,
-    )
-
-    with open(example_doc_path("simple.doc"), "rb") as f:
-        elements = partition_doc(file=f)
-
-    assert all(e.metadata.last_modified is None for e in elements)
-
-
-def test_partition_doc_from_file_pulls_last_modified_from_file_when_date_from_file_obj_arg_is_True(
-    mocker: MockFixture,
-):
-    modified_date_on_file = "2024-05-01T09:24:28"
-    mocker.patch(
-        "unstructured.partition.doc.get_last_modified_date_from_file",
-        return_value=modified_date_on_file,
-    )
-
-    with open(example_doc_path("simple.doc"), "rb") as f:
-        elements = partition_doc(file=f, date_from_file_object=True)
-
-    assert all(e.metadata.last_modified == modified_date_on_file for e in elements)
-
-
-def test_partition_doc_from_file_gets_None_last_modified_when_file_has_no_last_modified():
-    with open(example_doc_path("simple.doc"), "rb") as f:
-        sf = tempfile.SpooledTemporaryFile()
-        sf.write(f.read())
-        sf.seek(0)
-        elements = partition_doc(file=sf, date_from_file_object=True)
-
-    assert all(e.metadata.last_modified is None for e in elements)
-
-
-def test_partition_doc_from_file_prefers_metadata_last_modified_when_provided(mocker: MockFixture):
```

Comment:
So, these are integration tests that aren't needed any more since we have the unit tests for `get_last_modified`, right? I just want to double-check that we aren't losing coverage of any kind of conditions, though since this one and `test_partition_doc_from_file_suppresses_last_modified_from_file_by_default` sound like they're testing some extra conditions.

Reply:
That was my rationale, yes.

1. These fine-grained behaviors are not implemented here so we can't easily test them.
2. These behaviors are all now tested in `test_common.py`.

Less decisive but also a consideration are:
- These tests are repeated copypasta for almost all partitioners. We should be reducing that if we can.